api/routes.js

```diff
@@ -89,11 +89,11 @@ module.exports = function createRoutes(server) {
           if (err.notFound) code = 404
           if (err.statusCode) code = err.statusCode
           res.statusCode = code
-          response.json({status: 'error', error: err.message}).pipe(res)
+          response.json({status: 'error', message: err.message}).pipe(res)
           return
         }
 
-        if (!data) data = {status: 'error', error: 'no data returned'}
+        if (!data) data = {status: 'error', message: 'no data returned'}
```

Comment:
@maxogden from now on lets make sure we have a `message` property if we return errors to the frontend